lib/http.c

```diff
@@ -895,7 +895,9 @@ CURLcode Curl_http_input_auth(struct Curl_easy *data, bool proxy,
   /*
    * This resource requires authentication
    */
+#ifndef CURL_DISABLE_CRYPTO_AUTH
   struct connectdata *conn = data->conn;
+#endif
```

Comment:
Couldn't this rather be moved into the `#ifdef USE_SPNEGO` condition below? It would also match better with the actual use of the `negstate` variable which also depends on `USE_SPNEGO`.

Reply:
Unfortunately, it's also used in the `#ifndef CURL_DISABLE_CRYPTO_AUTH` block below for Digest.